File: Famoser.MassPass.Data/Models/ApiConfiguration.cs
using System;

namespace Famoser.MassPass.Data.Models
{
    public class ApiConfiguration
    {
        public Uri Uri { get; set; }
        public int Version { get; set; }
        public byte[] GenerationSalt { get; set; }
        public int GenerationKeyLenghtInBytes { get; set; }
        public int GenerationKeyInterations { get; set; }
        public byte[] InitialisationVector { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/ApiConfiguration.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 07/06/2016
 * Time: 18:19
 */

namespace Famoser\MassPass\Models;


class ApiConfiguration
{
    public $Uri;
    public $Version;
    public $GenerationSalt;
    public $GenerationKeyLenghtInBytes;
    public $GenerationKeyInterations;
    public $InitialisationVector;
}